apps/backend/src/deposits/cash-deposit.service.ts

```diff
@@ -60,25 +60,25 @@ export class CashDepositService {
       where: {
         pt_location_id,
         metadata: { program: program },
-        deposit_date: LessThanOrEqual(date),
+        deposit_date: date,
         status: status
       },
       order: {
-        deposit_date: 'ASC',
+        deposit_date: 'DESC',
```

Comment:
Keeping consistency with the order. Cash Match will still run in ascending order from the specified fiscal start -> fiscal_close, but the queries for payments and deposits will be ordered in descending order from the deposit date which is nearest (less than or equal) to the current reconciliation event date